docs/articles/expensify-classic/get-paid-back/expenses/Create-Expenses.md

```diff
@@ -88,12 +89,13 @@ Click the **Undelete** button and you're all set. You’ll find the expense on y
 ## What are the different Expense statuses?
 
 There are a number of different expense statuses in Expensify:
-1. **Unreported**: Unreported expenses are not yet part of a report (and therefore unsubmitted) and are not viewable by anyone but the expense creator/owner.
-2. **Open**: Open expenses are on a report that's still in progress, and are unsubmitted. Your Policy Admin will be able to view them, making it a collaborative step toward reimbursement.
+1. **Personal**: Personal expenses are not yet part of a report (and therefore unsubmitted) and are not viewable by anyone but the expense creator/owner.
+2. **Draft**: Draft expenses are seen as still in progress, and are unsubmitted. Your Policy Admin will be able to view them, making it a collaborative step toward reimbursement.
```

Comment:
`making it a collaborative step toward reimbursement` What does `it` refer to here? This line reads a little funky

Reply:
Haha someone from Marketing added that in...I also think it reads a little oddly.

Reply:
Tweaked this!